portability-gateway/src/main/java/org/dataportabilityproject/gateway/ApiMain.java

```diff
@@ -168,6 +187,7 @@ public TypeManager getTypeManager() {
     private final TypeManager typeManager;
     private final JobStore jobStore;
     private final SymmetricKeyGenerator keyGenerator;
+    private final List<AuthServiceExtension> extensions;
```

Comment:
can you rename this to authServiceExtensions?

Reply:
Done